Makefile

```diff
@@ -1227,6 +1230,21 @@ horizon_kuttl: namespace input openstack_crds openstack_storage_crds deploy_clea
 	make cleanup
 	make infra_cleanup
 
+.PHONY: openstack_kuttl_run
+openstack_kuttl_run: ## runs kuttl tests for the openstack operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${OPENSTACK_KUTTL_CONF} ${OPENSTACK_KUTTL_DIR}
+
+.PHONY: openstack_kuttl
+openstack_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy keystone rabbitmq keystone_deploy rabbitmq_deploy openstack openstack_deploy ## runs kuttl tests for the openstack operator. Installs openstack crds and mariadb, keystone, rabbitmq, ovn, infra and openstack operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+	$(eval $(call vars,$@,openstack))
+	make wait
+	make openstack_kuttl_run
+	make rabbitmq_deploy_cleanup
+	make deploy_cleanup
+	make openstack_cleanup
+	make keystone_cleanup
+	make mariadb_cleanup
```

Comment:
RabbitMQ, MariaDB and Keystone operators are all installed by the OpenStack operator itself.  So they should not be listed here as dependency targets nor should we call their cleanup targets.

Reply:
Thank you, updated.